### PR TITLE
Merge Pro_LoadTestV26 into catalyst

### PR DESCRIPTION
**Pull request by emranemran**
Bring in latest load test features/updates into catalyst.

### CODE REVIEW

**Comment by emranemran** [comment]
The only merge conflicts were a few minor ones in CMakeLists.txt

File: CMakeLists.txt
Comment (emranemran) [inline CMakeLists.txt]: FYI: removed this unused reference to .headers to fix build errors.

**Review by iameli — APPROVED** [review]
Cool, looks good assuming CI passes

File: CMakeLists.txt
Comment (emranemran) [reply CMakeLists.txt]: I'll revisit and try to solve this libsrt dependency correctly after merging this to unblock.

File: lib/dtsc.cpp
Comment (emranemran) [reply lib/dtsc.cpp:2201]: Do you mean break at compile time or run time?

Comment (iameli) [reply lib/dtsc.cpp:2201]: Compile. Looks like this builds tho

**Pull request merged**
